src/components/FilterList/filter-list.scss

```diff
@@ -1,11 +1,12 @@
-@import "../../css/variables.scss";
+@import "../../scss/variables.scss";
```

Comment:
hm, I think it's a little bit more common to name styles folder as `styles` without any specification to exact css pre-pro cessor that was used